2a. React/2. Mocking/src/mocks/handlers.ts

```diff
@@ -0,0 +1,47 @@
+import { http, HttpResponse } from 'msw';
+
+interface Project {
+    value: string;
+    label: string;
+}
+
+interface ProjectsResponse {
+    projects: Project[];
+    total: number;
+    filters: {
+        search: string | null;
+    };
+}
+
+const mockProjects: Project[] = [
```

Comment:
I feel like it would be better to either import a list from here to the test file or extract the demo list to a helper file and use in both places, but duplicating the exact same test list could cause problems if such approach was used in multiple places across many files for testing various api functions in a larger project. All in all, mock projects shouldn't be both here and in the test file